neo4j/test-integration/bookmark_test.go

```diff
@@ -346,8 +342,10 @@ var _ = Describe("Bookmark", func() {
 
 			bookmark = createNodeInTx(driver)
 
-			session, err = driver.Session(neo4j.AccessModeWrite, bookmark+"0")
-			Expect(err).To(BeNil())
+			session = driver.NewSession(neo4j.SessionConfig{
```

Comment:
Just want to confirm that you are no longer wanting to check against an error variable being null and that another mechanism is in use.

Reply:
the deprecated `driver.Session` function used to return an error, but the replacement `driver.NewSession` function does not return any error anymore, so there is no error to check